README.md

```diff
@@ -56,11 +56,14 @@ CRON=true /path/to/backup/script/backup-example.sh
 ```
 ## Installation
 
+There are multiple ways to install the role. Either clone or download it directly from the [github repository](https://github.com/roles-ansible/ansible_role_restic.git). Or Install it via [ansible galaxy](https://galaxy.ansible.com/do1jlr/restic):
 ```bash
-ansible-galaxy install arillso.restic
+ansible-galaxy install do1jlr.restic
 ```
 ## Requirements
 * bzip2
+* jmespath
```

Comment:
What is `jmespath` used for?

When I `grep` the project for this string, there are no matches, when I search the git history for it, the only mention I can find is that's being added to the README.